src/Event/Http/FpmHandler.php

```diff
@@ -179,6 +179,8 @@ private function eventToFastCgiRequest(HttpRequestEvent $event): ProvidesRequest
         $request->setServerPort($event->getServerPort());
         $request->setCustomVar('PATH_INFO', $event->getPath());
         $request->setCustomVar('QUERY_STRING', $event->getQueryString());
+        $request->setCustomVar('LAMBDA_INVOCATION_CONTEXT', json_encode($context));
+        $request->setCustomVar('LAMBDA_REQUEST_CONTEXT', json_encode($event->getRequestContext()));
         $request->setCustomVar('LAMBDA_CONTEXT', json_encode($event->getRequestContext()));
```

Comment:
@mnapoli Unsure what the best way to mark this as deprecated would be?

Reply:
I don't know what's best either ^^

Here might be a start, and I'll add that to the release notes as well.

```suggestion
        /** @deprecated The LAMBDA_CONTEXT variable is deprecated in favor  of LAMBDA_REQUEST_CONTEXT */
        $request->setCustomVar('LAMBDA_CONTEXT', json_encode($event->getRequestContext()));
```